ui.R

```diff
@@ -153,6 +153,9 @@ ui <- dashboardPage(
         fluidRow(
           infoBoxOutput(outputId = "infobox_party_en", width = NULL),
           infoBoxOutput(outputId = "infobox_constituency_en", width = NULL)
+        ),
+        fluidRow(
+          uiOutput("frame") # iframe
```

Comment:
Currently mis-aligned. Can indent with the following.

```suggestion
         # indent with tags$ul
          tags$ul(uiOutput(outputId = "frame", inline = TRUE)) # iframe
```